js/packages/teams-ai/src/planners/LLMClient.ts

```diff
@@ -325,6 +327,12 @@ export class LLMClient<TContent = any> {
                 return;
             }
 
+            // Ignore tool calls
+            // - see the tool call note below to understand why we're ignoring them.
+            if ((chunk.delta as any)?.tool_calls || chunk.delta?.action_calls) {
+                return;
```

Comment:
to confirm- are we returning here because the model will only return either streamed chunks **or** tool calls, e.g., never both?

Reply:
^ actually this doesn't seem to be the case if we are clearing it in L368- which means that we won't be streaming any text chunks until the tool calls have returned

from a UX standpoint, have you seen this result in a long buffer time with the info message if there are multiple tool calls requested?